### PR TITLE
Refactor to move Scene-specific settings out of DataController

### PR DESCRIPTION
**Pull request by dstrawberrygirl**
SimulationManagerComponent instance on each scene now sets up each scene with stars, markers, constellations.

Use SimulationManager singleton to maintain references to all core components
Pass updates to each component when scenes load
Remove old Celestial Sphere prefabs from all scenes, now only instantiate once from a SimulationManagerComponent instance on a scene and persist across scenes

This needs some testing! @mklewandowski I saw some strange issues relating to constellation connections apparently not rescaling after switching scenes and changing the number of stars visible with the magnitude slider, but I want to confirm if this is a result of recent changes or a bug from before the refactor.

### CODE REVIEW

**Comment by mklewandowski** [comment]
The number of visible stars is an existing issue.  If the visible stars were changed in the Stars scene via the magnitude slider, then those changes would persist in the next loaded scene.  I think that is correct, however, we should ideally add the slider to views with Stars so it can be adjusted at any point.
Not sure that there are any paths where the constellation connections rescaled from view to view.  So what you're seeing there most likely already existed.

File: Assets/Scripts/DataController.cs [hunk 1]
```diff
@@ -93,6 +68,26 @@ public DateTime CurrentSimulationTime
...
+    private bool showHorizonView = false;
+    private float simulationTimeScale = 10f;
+    private float radius = 50;
+    public float Radius
+    {
+        get { return radius; }
+        private set { radius = value; }
+    }
+
+    private bool isReady = false;
```
Comment (mklewandowski) [inline Assets/Scripts/DataController.cs]: Were there issues tied to moving the sphere and scene loading?

Comment (dstrawberrygirl) [reply Assets/Scripts/DataController.cs]: isReady was added mainly to stop fixed update from running until we've reset things

**Comment by dstrawberrygirl** [comment]
thanks for the feedback! Yes I'll add scene-specific settings for markers and lines to the manager component. 

Stars static - not sure how it broke but I will fix it - I do want to keep it in case we just want to generate a sphere of stars without all the star components, but I'll move it to a sub folder.

**Pull request merged**